Codeinwp/neve
# feat: autoactivate TPC from welcome notice

### Summary

I have replaced the button from the welcome notice ("Try one of our ready to use Starter Sites") with a React component button.
The new component handles installing TPC, activating TPC, or dismissing the welcome notice. Based on the state of the TPC plugin.

Cases:
1. If not present it will install, activate and then dismiss the notice.
2. If present but not active will, activate and then dismiss the notice.
3. If present and active will dismiss the notice.

After any of the cases is run it will also redirect the user to the starter sites onboarding.

### Will affect the visual aspect of the product

NO

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/201977557-9e4602cb-58f1-411b-a4b0-44276fd227c0.png)

![image](https://user-images.githubusercontent.com/23024731/201977678-920f1411-d8aa-4b75-b4e9-5cbc3a335737.png)

![image](https://user-images.githubusercontent.com/23024731/201977821-4390c6bf-d132-4133-82ec-bfbaced4e67c.png)



### Test instructions

1. On a clean instance, check ( Case 1 )
2. Reset the instance (or set the `neve_notice_dismissed` option to `no`), have TPC installed but not active, and check ( Case 2 )
3.  Reset the instance (or set the `neve_notice_dismissed` option to `no`), check ( Case 3 )


Closes #3287.
